Configure.cmake

```diff
@@ -42,6 +45,28 @@ CHECK_C_SOURCE_COMPILES("
     __m128d r = _mm_mul_pd(_mm_set1_pd(1), _mm_set1_pd(2)); }"
   COMPILER_SUPPORTS_SSE2)
 
+set (CMAKE_REQUIRED_FLAGS ${FLAGS_ENABLE_SSE4})
+CHECK_C_SOURCE_COMPILES("
+  #if defined(_MSC_VER)
+  #include <intrin.h>
+  #else
+  #include <x86intrin.h>
+  #endif
+  int main() {
+    __m128d r = _mm_floor_sd(_mm_set1_pd(1), _mm_set1_pd(2)); }"
+  COMPILER_SUPPORTS_SSE4)
+
+set (CMAKE_REQUIRED_FLAGS ${FLAGS_ENABLE_AVX})
```

Comment:
This would require to add the `try_run` checks that I mentioned in https://github.com/shibatch/sleef/pull/45#discussion_r132029105

I am fine doing that, only I would like to avoid as much as possible checks in the form "if compiler is [whatever]", as I prefer to ask questions in the form "does compiler supports [feature]" or "does target can run [feature]", as they are target and compiler independent.